src/Altinn.Platform/Altinn.Platform.Storage/Storage/Program.cs

```diff
@@ -56,6 +56,10 @@ public static void Main(string[] args)
 
                 logging.AddProvider(new SerilogLoggerProvider(logger));
             })
+            .UseKestrel()
```

Comment:
Remember! These lines are to be used only in local development. So this should be commented out